packages/grid/data-grid/src/DataGrid.test.tsx

```diff
@@ -4,23 +4,26 @@ import PropTypes from 'prop-types';
 // @ts-ignore
 import { createClientRender } from 'test/utils';
 import { expect } from 'chai';
-import { DataGrid } from '@material-ui/data-grid';
+import { DataGrid, RowsProp, Columns } from '@material-ui/data-grid';
 
 describe('<DataGrid />', () => {
-  const render = createClientRender();
-
-  const defaultProps = {
-    rows: [
-      {
-        id: 0,
-        brand: 'Nike',
-      },
-    ],
-    columns: [
-      { field: 'id', hide: true },
-      { field: 'brand', width: 100 },
-    ],
-  };
+  let render;
+  let defaultProps;
+  beforeEach(() => {
+    render = createClientRender();
```

Comment:
createClientRender() is already wrapping
a beforeEach, it should stay at the top level.